File: go/1044.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var A int
	var B int

	fmt.Scan(&A)
	fmt.Scan(&B)

	if A%B == 0 || B%A == 0 {
		fmt.Println("Sao Multiplos")
	} else {
		fmt.Println("Nao sao Multiplos")
	}
}

File: Python/1 - Iniciante/1044.py
# coding: UTF - 8
# Funcional no URI

A, B = input().split(" ")
A = int(A)
B = int(B)
if A % B == 0 or B % A == 0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")